scripts/fetch/fetchGA.R

```diff
@@ -7,6 +7,10 @@ library(assertthat)
 library(aws.s3)
 library(aws.signature)
 
+fetchTimestamp.GAviews <- function(viz) {
+  invisible()
```

Comment:
This will never create a timestamp file, and therefore the `GAviews` fetcher will always fetch, on every build. Is that what you want?

Reply:
I think so... @wdwatkins is that what we want?

Reply:
Might be nice for it to not run more than once in a day.

Reply:
Although maybe the distinction between fetching from S3 vs hitting google could complicate things.